cpp/dolfinx/fem/interpolate.h

```diff
@@ -618,27 +695,14 @@ void interpolate_nonmatching_meshes(
       valuesT(j, i) = values(i, j);
 
   // Call local interpolation operator
-  fem::interpolate<T>(u, std::span(valuesT_b.data(), valuesT_b.size()),
-                      {valuesT.extent(0), valuesT.extent(1)}, cells);
+  fem::interpolate<T>(u, valuesT_b, {valuesT.extent(0), valuesT.extent(1)},
+                      cells);
 }
 //----------------------------------------------------------------------------
 } // namespace impl
 
-/// Interpolate an expression f(x) in a finite element space
-///
-/// @param[out] u The function to interpolate into
-/// @param[in] f Evaluation of the function `f(x)` at the physical
-/// points `x` given by fem::interpolation_coords. The element used in
-/// fem::interpolation_coords should be the same element as associated
-/// with `u`. The shape of `f` should be (value_size, num_points), with
-/// row-major storage.
-/// @param[in] fshape The shape of `f`.
-/// @param[in] cells Indices of the cells in the mesh on which to
-/// interpolate. Should be the same as the list used when calling
-/// fem::interpolation_coords.
-/// @tparam Scalar type
-template <typename T>
-void interpolate(Function<T>& u, std::span<const T> f,
+template <typename T, typename U>
+void interpolate(Function<T, U>& u, std::span<const T> f,
```

Comment:
Did you mean to move the docs into the implicit namespace, which is not in the online docs?

Reply:
It's documented (where declared nearer top of file) and appears in docs.